scss/navbar_hover.css

```diff
@@ -0,0 +1,46 @@
+@import "https://fonts.googleapis.com/css?family=Raleway";
+
+div.container a {
+  color: rgb(24, 187, 24);
+  text-decoration: none;
+  font: 20px Raleway;
+  margin: 0px 10px;
+  padding: 10px 10px;
+  position: relative;
+  z-index: 0;
+  cursor: pointer;
+}
+
+.gray {
+  background: none;
+}
+
+div.highlightTextOut a:before,
+div.highlightTextIn a:before {
+  position: absolute;
+  /*color: #FFF;*/
+  color: lightgreen;
+  top: 0px;
+  left: 0px;
+  padding: 10px;
+  overflow: hidden;
+  content: attr(alt);
+  transition: all 0.5s;
+  transform: scale(0.8);
+  opacity: 0;
+}
+
+div.highlightTextOut a:hover:before,
+div.highlightTextIn a:hover:before {
+  transform: scale(1);
+  opacity: 1;
+}
+
+/* Highlight text in  */
+div.highlightTextIn a {
+  color: white;
+}
+
+div.highlightTextIn a:before {
+  transform: scale(1.5);
+}
```

Comment:
make all these CSS changes in _header.scss file

Reply:
There Seems to be a issue while compiling .scss file, The changes are not being reflected in the live server. For now we will go with external css styling (which works fine), Once the issue is resolved I'll Issue a new PR for the same.

Reply:
You need to install **Sass compiler (by Ritwick Dey)**, the VS Code extension to compile the .scss file.